README.md

```diff
@@ -3,6 +3,11 @@
 - Testing future versions of UI for [stampy.ai](https://stampy.ai/read/Get_involved)
 - Using [Remix](https://remix.run/docs) and [Cloudflare Workers](https://developers.cloudflare.com/workers)
 
+- Install Node, npm
+- npx create-remix@1.4.3
+- Replace 'remix' with '@remix-run/cloudflare' or '@remix-run/cloudflare/react'
+- Install react-auto-height, copy-to-clipboard, node-html-parser
```

Comment:
Didn't know to do/try that. I should have mentioned or emphasized (and it is probably obvious)... it's been a long time since I worked in development so I definitely appreciate tips/guidance on best practices nowadays.